### DISCUSSION TITLE
Little question about setpoint command

### DISCUSSION THREAD

**Discussion by Omikaa** [bitcraze/discussions#1458] [Q&A]
![image](https://github.com/bitcraze/crazyflie-lib-python/assets/136170290/042bc97c-baef-43db-9da4-1a0dbd95709d)
There is a little question about the command above.
What is the range of the attitude part about command? The max degree of  roll or pitch is 2*pi?  What about the max angular velocity of yaw?

**Comment by ToveRumar** [comment]
Hi!

Regarding the angular velocity of the yawrate there is no defined max except it needs to fit in a 4 byte floating number.  If you sent a set point with a really high yawrate i guess you would get issues controlling the drone. 

If your question is more about decimal precision we don't define that either, I guess it could be an interesting thing to test though. 

Regarding max of roll/pitch. This is in degrees (not radians), and there is no defined max here either, except it should also fit in a 4 byte float. 
Although your drone would crash if you set it so high it does not have enough upward thrust to have lift. This  depends on the pitch/roll angle and the amount of thrust you give it. 

These answers are a bit fluffy, since they depend on the actual flight dynamics, and what the application is. 

I hope this answers your questions!
